argilla/src/argilla/datasets/_export/card/_parser.py

```diff
@@ -0,0 +1,35 @@
+#  Copyright 2021-present, the Recognai S.L. team.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+# TODO: This license is not consistent with the license used in the project.
+#       Delete the inconsistent license and above line and rerun pre-commit to insert a good license.
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+AVAILABLE_SIZE_CATEGORIES = {
```

Comment:
maybe we can import this from the Hugging Face hub library?

Reply:
We discussed adding this as lightweight core dependency because of telemetry too.

Reply:
this doesn't exist in `huggingface_hub` but could be added if relevant. Do you know if `datasets` has a similar mapping?